pkg/yum/repository.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 // Max uncompressed XML file supported
-const DefaultMaxXmlSize = int64(512 * 1024 * 1024) // 512 MB
```

Comment:
Should we make it a little higher, just in case it creeps up again? Maybe double it to 1024 MB

Reply:
yea i was thinking that at first but wasn't sure if it was overkill, but i agree! i'll increase it :)